### PR TITLE
Content layout

### PR DESCRIPTION
**Pull request by rodrigoWolox**
## Summary
- Content layout card, first commit. Im already installing peek so i can send you a gif of the layout, meanwhile i leave two pics ;)

## Screenshot
content-layout3

## Trello Card
https://trello.com/c/EjeJ8DMS/23-content-layout

### CODE REVIEW

**Comment by rodrigoWolox** [comment]
I leave you a little gif from the layout content, hope you like it!

content-layout

File: index.html [hunk 1]
```diff
@@ -9,7 +9,84 @@
   <link href="https://fonts.googleapis.com/css?family=Roboto" rel="stylesheet">
   <link rel="stylesheet" href="application.css" type="text/css">
 </head>
-<body>
-  <h1 class="page-title">Front-End Tools</h1>
+<body class="container">
+  <h1 class="text-big page-title">Front-End Tools</h1>
```
Comment (mmamaniwolox) [inline index.html]: It is supposed to be only one `<h1>` in a page. Now that you have two titles for technologies, you shouldn't be using `<h1>`.  h2 may be fine

File: index.html [hunk 2]
```diff
@@ -9,7 +9,84 @@
...
+    <div class="tech-container">
+      <img class="tech-logo" src="assets/react_icon.svg" alt="ReactJS logo">
+      <p class="tech-title">ReactJS</p>
+      <p class="tech-sub">Release: 2013</p>
+      <p class="tech-desc">
+        React is declarative, component-based Javascript library for building user interfaces.
+      </p>
+    </div>
+    <div class="tech-container">
+      <img class="tech-logo" src="assets/angular_icon.svg" alt="ReactJS logo">
```
Comment (mmamaniwolox) [inline index.html]: Change alt, in this one and all the others

File: index.html [hunk 3]
```diff
@@ -9,7 +9,84 @@
...
+    <div class="tech-container">
+      <img class="tech-logo" src="assets/polymer_icon.svg" alt="ReactJS logo">
+      <p class="tech-title">Polymer</p>
+      <p class="tech-sub">Release: 2015</p>
+      <p class="tech-desc">
+        Polymer is an open-source JavaScript library for building web applications using Web Components.
+      </p>
+    </div>
+  </div>
+  <h1 class="text-big page-title">Back-End Tools</h1>
```
Comment (mmamaniwolox) [inline index.html]: Same than before, use h2

File: scss/tech.scss [hunk 4]
```diff
@@ -1,3 +1,56 @@
...
+
+.tech-container {
+  background-color: $earls-green;
+  border-radius: 5px;
+  height: auto;
+  margin: 20px;
+  padding: 40px;
+  position: relative;
+  text-align: center;
+  width: 300px;
```
Comment (mmamaniwolox) [inline scss/tech.scss]: I think this is too much for what I see in your screen and the one in trello

File: scss/tech.scss [hunk 5]
```diff
@@ -1,3 +1,56 @@
...
+
+  &::before {
+    border: 1px solid;
+    border-color: $earls-green;
+    border-radius: 5px;
+    content: '';
+    height: 100%;
+    right: 15px;
+		position: absolute;
```
Comment (mmamaniwolox) [inline scss/tech.scss]: Indentation is wierd here

File: scss/tech.scss [hunk 6]
```diff
@@ -1,3 +1,56 @@
...
+  margin: 10px;
+}
+
+.tech-sub {
+  font-weight: 900;
+  margin: 20px;
+}
+
+.tech-desc {
+  margin: 30px 15px 25px;
```
Comment (mmamaniwolox) [inline scss/tech.scss]: You shouldn't need this margins, what are they for?

File: scss/tech.scss [hunk 7]
```diff
@@ -1,3 +1,56 @@
...
+  flex-wrap: wrap;
+  margin: 50px 0px 150px;
+}
+.tech-title {
+  margin: 10px;
+}
+
+.tech-sub {
+  font-weight: 900;
+  margin: 20px;
```
Comment (mmamaniwolox) [inline scss/tech.scss]: This margin is adding to the one from tech-title and you dont need it in the sides, better to use just margin-bottom

**Review by mmamaniwolox — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (rodrigoWolox) [reply index.html]: Do i leave "Front-end tools" in h1 tag and "Back-End tools" in h2 tag?

Comment (mmamaniwolox) [reply index.html]: No, both h2

File: index.html (hunk 2 above)
Comment (rodrigoWolox) [reply index.html]: What would be a good alt text? Something more descriptive as "Error loading ReactJS logo, sorry :(" i dont know

Comment (mmamaniwolox) [reply index.html]: You have repeated `React JS logo" in all of them, just that

File: scss/tech.scss (hunk 4 above)
Comment (rodrigoWolox) [reply scss/tech.scss]: I agree, padding: 30px; width: 280px; seems better. 

image

If i put less of that 5 tech containers jump in the same row.

Comment (mmamaniwolox) [reply scss/tech.scss]: It doesn't mind how many techs do you have in the same row :P

File: scss/tech.scss (hunk 5 above)
Comment (rodrigoWolox) [reply scss/tech.scss]: Don't know what is going on. I have it ok in my file. 

image

File: scss/tech.scss [hunk 8]
```diff
@@ -1,3 +1,56 @@
...
+		position: absolute;
+		bottom: 15px;
+		width: 100%;
+		z-index: -1;
+  }
+}
+
 .tech-logo {
   height: 100px;
+  width: 100px;
```
[the remark this answers is not in the document]
Comment (rodrigoWolox) [reply scss/tech.scss]: Yup!

image

[the remark this answers is not in the document]
Comment (mmamaniwolox) [reply scss/tech.scss]: Check removing the height then

File: scss/tech.scss (hunk 6 above)
Comment (rodrigoWolox) [reply scss/tech.scss]: I use them to shape the text by encapsulating it in a smaller box, i could put some more padding in the container or delete this margins and let the world burn :V

Comment (mmamaniwolox) [reply scss/tech.scss]: lol, I think container padding should be enought

File: index.html (hunk 1 above)
Comment (rodrigoWolox) [reply index.html]: Ok, done!

File: index.html (hunk 2 above)
Comment (rodrigoWolox) [reply index.html]: Ohhhhhhhh, im so sorry, really. Consider it done!

File: scss/tech.scss (hunk 4 above)
Comment (rodrigoWolox) [reply scss/tech.scss]: Ok, get it!

File: scss/tech.scss [hunk 9]
```diff
@@ -1,3 +1,53 @@
+@import 'colors';
+@import 'fonts';
+
+.tech-container {
+  background-color: $earls-green;
+  border-radius: 5px;
+  height: auto;
```
Comment (mmamaniwolox) [inline scss/tech.scss]: I think this is default prop, you shouldnt need this

File: scss/tech.scss [hunk 10]
```diff
@@ -1,3 +1,53 @@
...
+  margin: 20px;
+  padding: 10px 20px 20px 10px;
```
Comment (mmamaniwolox) [inline scss/tech.scss]: Order is `top right bottom left`. Do you want content to be moved top left? If not, `10px 20px` should work

File: scss/tech.scss [hunk 11]
```diff
@@ -1,3 +1,53 @@
...
+  &::before {
+    bottom: 10px;
+    border: 1px solid;
+    border-color: $earls-green;
+    border-radius: 5px;
+    content: '';
+    height: 100%;
+    position: absolute;
+    right: 10px;
+		width: 100%;
```
Comment (mmamaniwolox) [inline scss/tech.scss:22]: Almost there with indentation >.<

**Review by mmamaniwolox — CHANGES_REQUESTED** [review]

File: scss/tech.scss (hunk 9 above)
Comment (rodrigoWolox) [reply scss/tech.scss]: You are right!

File: scss/tech.scss (hunk 11 above)
Comment (rodrigoWolox) [reply scss/tech.scss:22]: I dont know whats going on with this, in my file is ok. Will try something, hope it works.

File: scss/tech.scss (hunk 10 above)
Comment (rodrigoWolox) [reply scss/tech.scss]: It was just to move away the bottom and right a little more like is in the card's image. I set all the padding to 20px now, let me know if you are ok with it.

**Review by mmamaniwolox — APPROVED** [review]

**Pull request merged**